SU2_CFD/include/drivers/CDriver.hpp

```diff
@@ -467,19 +467,29 @@ class CDriver : public CDriverBase {
   /*!
    * \brief Process the boundary conditions and update the multi-grid structure.
    */
-  void BoundaryConditionsUpdate();
+  void UpdateBoundaryConditions();
```

Comment:
No renaming please, this breaks compatibility.

Reply:
I will revert the changes for now, however they should be updated in a future PR for consistency

Reply:
version 9